src/sdk/lsf-sdk.js

```diff
@@ -563,7 +563,12 @@ export class LSFWrapper {
         { body },
         // don't react on duplicated annotations error
         { errorHandler: result => result.status === 409 },
-      );
+      ).then(result => {
```

Comment:
promise chain is broken here because it doesn't pass `result` through. I'd remove current `return`s and added final `return result` the same way as in "update" version.

Reply:
good catch, thanks!